Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/WMB.hs

```diff
@@ -61,7 +309,19 @@ postUiWmbTripStart ::
     API.Types.UI.WMB.TripStartReq ->
     Environment.Flow Kernel.Types.APISuccess.APISuccess
   )
-postUiWmbTripStart = do error "Logic yet to be decided"
+postUiWmbTripStart (person, _, city) tripTransactionId obj = do
```

Comment:
name it personId instead of person also `obj` -> `req` ([good read on proper naming](https://dev.to/gervaisamoah/a-guide-to-clean-code-the-power-of-good-names-3f6i?utm_source=chatgpt.com))